app/src/main/java/org/oppia/android/app/settings/profile/ProfileRenameFragmentPresenter.kt

```diff
@@ -101,6 +101,7 @@ class ProfileRenameFragmentPresenter @Inject constructor(
   }
 
   private fun handleAddProfileResult(result: AsyncResult<Any?>, profileId: Int) {
+    val profileId = ProfileId.newBuilder().apply { internalId = profileId }.build()
```

Comment:
Since internalId is being accessed above, could profileId just be passed in here to avoid creating a new ProfileId?

Reply:
Yeah Ben sure updated.